src/tests/test-utils.ts

```diff
@@ -71,3 +74,15 @@ export class GlobalTestState {
     return null!;
   }
 }
+export const storageMock = {
+  getItem: (storageMap: {
```

Comment:
shouldn't this be called `setItem`?

Reply:
This is mocking`getItem` of storage as you can see in line 84 hence named it `getItem`

Reply:
How about https://github.com/excalidraw/excalidraw/pull/2313/commits/e6780a7fee965f064e163763ce0fcaccb1f62f61 ?.  Mocked the local storage API so it can be used easily with tests (will make it typesafe).